src/components/AirTicketsSearch/AirTicketsSearch.component.js

```diff
@@ -0,0 +1,187 @@
+import React, { useState, useEffect } from 'react';
+import DateRangePicker from 'react-bootstrap-daterangepicker';
+import 'bootstrap-daterangepicker/daterangepicker.css';
+
+import ItineraryModalContainer from '../ItinerarayModalContainer/ItineraryModalContainer.component';
+import PassengSubContent from '../PassengerSubContent/PassengerSubContent.component';
+
+import {
+	AirTicketsSearchContainer,
+	SearchContainer,
+} from './AirTicketsSearch.styles';
+
+const AirTicketsSearch = () => {
+	const [passengerSubContentHidden, setPassengerSubContentHidden] = useState(
+		true,
+	);
+	const [adultAmount, setAdultAmount] = useState(0);
+	const [childAmount, setChildAmount] = useState(0);
+	const [babyAmount, setBabyAmount] = useState(0);
+	const [departure, setDeparture] = useState('김포');
+	const [arrival, setArrival] = useState('제주');
+	const [modalHidden, setModalHidden] = useState(true);
+	const [seatSelection, setSeatSelection] = useState([
+		true,
+		false,
+		false,
+		false,
+	]);
+
+	const [locationTarget, setLocationTarget] = useState(null);
+
+	const changeLocation = (e) => {
+		if (locationTarget === departure) setDeparture(e.target.innerText);
+		else setArrival(e.target.innerText);
+		setModalHidden(!modalHidden);
+	};
+
+	const getSelection = (num) => {
+		let selectionArray = [false, false, false, false];
+		selectionArray[num] = true;
+		setSeatSelection(selectionArray);
+	};
+
+	const changeTargetAmount = (increase, targetAmount, targetFunction) => {
+		if (targetAmount === 0 && !increase) return;
+		increase
+			? targetFunction(targetAmount + 1)
+			: targetFunction(targetAmount - 1);
+	};
+
+	const totalAmount = () => {
+		return adultAmount + childAmount + babyAmount;
+	};
+
+	const checkSearchAvailability = () => {
+		!totalAmount() && alert('인원 수 문제');
+	};
+
+	const swapDepartureAndArrival = () => {
+		const temp = departure;
+		setDeparture(arrival);
+		setArrival(temp);
+	};
+
+	const changeDateResultToKR = (e) => {
+		const start = e.target.value.split(' - ')[0];
+		const end = e.target.value.split(' - ')[1];
```

Comment:
```suggestion
	const changeDateResultToKR = (e) => {
	  const [start, end] = e.target.value.split(' - ')
```
구조분해 할당 활용하면 더욱 간단하게 표현할 수 있습니다

Reply:
수정했습니다 😄